CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ permalink: /docs/en-US/changelog/
 * Always set the database root user password to avoid having the default invalid password on fresh installs
 * Don't spider and recurse domains when checking for network connections
 * Swap the MariaDB apt mirror used for a more reliable source
+* Not import the database if the site has `skip_provisioning` as true
```

Comment:
I'd say that the changelog would be "Do not restore the database for a site if the site has `skip_provisioning` as true"